modules/synacormediaBidAdapter.js

```diff
@@ -48,6 +48,13 @@ export const spec = {
       imp: []
     };
 
+    const callbackTimeout = bidderRequest.timeout;
+    const globalTimeout = config.getConfig('bidderTimeout');
+    const tmax = globalTimeout ? Math.min(globalTimeout, callbackTimeout) : callbackTimeout;
+    if (tmax) {
+      openRtbBidRequest.tmax = tmax;
```

Comment:
> This might be unwise; you probably want to shave off a good bit for transmission time and cap at some value

@patmmccann We currently do this in our adserver itself. We subtract from the tmax we receive based upon the source and cap it with a floor (I think it's currently 100ms). We do it in the adserver because we can adjust it outside of prebid releases.